src/errors.d

```diff
@@ -18,6 +18,8 @@ import ddmd.globals;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 
+//debug = printGaggedErrors;
```

Comment:
1. I don't need to scroll and find the line. I am already looking at the function, because I know it went wrong. In your scheme, I then have to scroll up going looking for where the version is defined.

2. If I put in a special logging version name for every commented out logging line, I'd have a lot more lines of code and zero added value.

I'm sorry, this change just does not seem worth while.